diff --git a/WORKSPACE b/WORKSPACE
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -53,6 +53,8 @@ python_init_repositories(
         "jaxlib*",
         "jax_cuda*",
         "jax-cuda*",
+        "jax_rocm*",
+        "jax-rocm*",
     ],
     local_wheel_workspaces = ["//jaxlib:jax.bzl"],
     requirements = {
diff --git a/jaxlib/jax.bzl b/jaxlib/jax.bzl
--- a/jaxlib/jax.bzl
+++ b/jaxlib/jax.bzl
@@ -200,9 +200,12 @@ def _gpu_test_deps():
             "//jax_plugins:gpu_plugin_only_test_deps",
         ],
         "//jax:config_build_jaxlib_false": if_cuda_is_configured([
-            "//jaxlib/tools:pypi_jax_cuda_plugin_with_cuda_deps",
-            "//jaxlib/tools:pypi_jax_cuda_pjrt_with_cuda_deps",
-        ]) + if_rocm_is_configured(EXTERNAL_DEPS),
+            #"//jaxlib/tools:pypi_jax_cuda_plugin_with_cuda_deps",
+            #"//jaxlib/tools:pypi_jax_cuda_pjrt_with_cuda_deps",
+        ]) + if_rocm_is_configured([
+            "//jaxlib/tools:rocm_plugin_kernels_wheel",
+            "//jaxlib/tools:rocm_plugin_pjrt_wheel",
+        ]),
         "//jax:config_build_jaxlib_wheel": if_cuda_is_configured([
             "//jaxlib/tools:jax_cuda_plugin_py_import",
             "//jaxlib/tools:jax_cuda_pjrt_py_import",
@@ -311,7 +313,7 @@ def jax_multiplatform_test(
             shard_count = test_shards,
             tags = test_tags,
             main = main,
-            exec_properties = tf_exec_properties({"tags": test_tags}),
+            exec_properties = {},  # tf_exec_properties({"tags": test_tags})
             visibility = jax_visibility(name),
             legacy_create_init = 0,
         )
diff --git a/jaxlib/tools/BUILD.bazel b/jaxlib/tools/BUILD.bazel
--- a/jaxlib/tools/BUILD.bazel
+++ b/jaxlib/tools/BUILD.bazel
@@ -477,6 +477,25 @@ py_import(
     wheel_deps = if_pypi_cuda_wheel_deps([":nvidia_wheel_deps"]),
 )
 
+filegroup(
+    name = "rocm_wheel_deps",
+    srcs = [
+    ],
+)
+
+# Targets for importing ROCm plugin wheels
+py_import(
+    name = "rocm_plugin_kernels_wheel",
+    wheel = "@pypi_jax_rocm7_plugin//:whl",
+    wheel_deps = [":rocm_wheel_deps"],
+)
+
+py_import(
+    name = "rocm_plugin_pjrt_wheel",
+    wheel = "@pypi_jax_rocm7_pjrt//:whl",
+    wheel_deps = [":rocm_wheel_deps"],
+)
+
 # Mosaic GPU
 
 py_binary(
