From 1c4ef35e332f671ba07ad7076bc8222a4a41314e Mon Sep 17 00:00:00 2001
From: Charles Hofer <Charles.Hofer@amd.com>
Date: Wed, 12 Nov 2025 21:42:44 +0000
Subject: [PATCH 2/3] Make jaxlib targets visible

---
 jaxlib/BUILD      | 2 +-
 jaxlib/rocm/BUILD | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/jaxlib/BUILD b/jaxlib/BUILD
index ddf852bff..d8ca0c784 100644
--- a/jaxlib/BUILD
+++ b/jaxlib/BUILD
@@ -39,7 +39,7 @@ licenses(["notice"])
 
 package(
     default_applicable_licenses = [],
-    default_visibility = ["//jax:internal"],
+    default_visibility = ["//visibility:public"],
 )
 
 package_group(
diff --git a/jaxlib/rocm/BUILD b/jaxlib/rocm/BUILD
index 4a2d4fad9..f59ac817a 100644
--- a/jaxlib/rocm/BUILD
+++ b/jaxlib/rocm/BUILD
@@ -27,7 +27,7 @@ licenses(["notice"])
 
 package(
     default_applicable_licenses = [],
-    default_visibility = ["//:__subpackages__"],
+    default_visibility = ["//visibility:public"],
 )
 
 cc_library(
-- 
2.34.1

