From 9ef3beaac2ceabf66b6af4282781b4be3685b6a3 Mon Sep 17 00:00:00 2001
From: Charles Hofer <Charles.Hofer@amd.com>
Date: Thu, 13 Nov 2025 23:24:23 +0000
Subject: [PATCH] Fix HIP availability errors

---
 jaxlib/rocm/BUILD | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/jaxlib/rocm/BUILD b/jaxlib/rocm/BUILD
index 4a2d4fad9..8915225ed 100644
--- a/jaxlib/rocm/BUILD
+++ b/jaxlib/rocm/BUILD
@@ -358,6 +358,7 @@ nanobind_extension(
         "//jaxlib:kernel_nanobind_helpers",
         "@local_config_rocm//rocm:hip_runtime",
         "@local_config_rocm//rocm:rocm_headers",
+	"@local_config_rocm//rocm:hip",
         "@nanobind",
     ],
 )
@@ -401,6 +402,7 @@ nanobind_extension(
         "@com_google_absl//absl/base",
         "@local_config_rocm//rocm:hip_runtime",
         "@local_config_rocm//rocm:rocm_headers",
+	"@local_config_rocm//rocm:hip",
         "@nanobind",
         "@xla//xla/ffi/api:ffi",
     ],
@@ -468,6 +470,7 @@ nanobind_extension(
         "@com_google_absl//absl/status",
         "@com_google_absl//absl/status:statusor",
         "@local_config_rocm//rocm:hip_runtime",
+	"@local_config_rocm//rocm:hip",
         "@nanobind",
     ],
 )
-- 
2.34.1

