diff --git a/jaxlib/jax.bzl b/jaxlib/jax.bzl
index 067e4dd6e..5e0319b45 100644
--- a/jaxlib/jax.bzl
+++ b/jaxlib/jax.bzl
@@ -192,10 +192,13 @@ def _gpu_test_deps():
             "//jaxlib/rocm:gpu_only_test_deps",
             "//jax_plugins:gpu_plugin_only_test_deps",
         ],
-        "//jax:config_build_jaxlib_false": [
+        "//jax:config_build_jaxlib_false": if_rocm_is_configured([
+            "//jaxlib/tools:rocm_plugin_kernels_wheel",
+            "//jaxlib/tools:rocm_plugin_pjrt_wheel",
+        ]) + if_cuda_is_configured([
             "//jaxlib/tools:pypi_jax_cuda_plugin_with_cuda_deps",
             "//jaxlib/tools:pypi_jax_cuda_pjrt_with_cuda_deps",
-        ],
+        ]),
         "//jax:config_build_jaxlib_wheel": [
             "//jaxlib/tools:jax_cuda_plugin_py_import",
             "//jaxlib/tools:jax_cuda_pjrt_py_import",
@@ -302,7 +305,8 @@ def jax_multiplatform_test(
             shard_count = test_shards,
             tags = test_tags,
             main = main,
-            exec_properties = tf_exec_properties({"tags": test_tags}),
+            exec_properties = {},  #tf_exec_properties({"tags": test_tags}),
+            legacy_create_init = 0,
         )
 
 def jax_generate_backend_suites(backends = []):
diff --git a/jaxlib/tools/BUILD.bazel b/jaxlib/tools/BUILD.bazel
index 3a1a48736..347b37091 100644
--- a/jaxlib/tools/BUILD.bazel
+++ b/jaxlib/tools/BUILD.bazel
@@ -476,6 +476,24 @@ py_import(
     wheel_deps = if_pypi_cuda_wheel_deps([":nvidia_wheel_deps"]),
 )
 
+filegroup(
+    name = "rocm_wheel_deps",
+    srcs = [
+    ],
+)
+
+py_import(
+    name = "rocm_plugin_kernels_wheel",
+    wheel = "@pypi_jax_rocm7_plugin//:whl",
+    wheel_deps = [":rocm_wheel_deps"],
+)
+
+py_import(
+    name = "rocm_plugin_pjrt_wheel",
+    wheel = "@pypi_jax_rocm7_pjrt//:whl",
+    wheel_deps = [":rocm_wheel_deps"],
+)
+
 # Mosaic GPU
 
 py_binary(
