diff --git a/jaxlib/gpu/BUILD b/jaxlib/gpu/BUILD
index 6169c882a..72f403038 100644
--- a/jaxlib/gpu/BUILD
+++ b/jaxlib/gpu/BUILD
@@ -119,6 +119,7 @@ cc_library(
         "-fno-strict-aliasing",
     ],
     features = ["-use_header_modules"],
+    visibility = ["//visibility:public"],
     deps = [
         "//jaxlib:kernel_nanobind_helpers",
         "@com_google_absl//absl/status",
diff --git a/jaxlib/rocm/BUILD b/jaxlib/rocm/BUILD
index 1b8e8dd1e..ffcbb664e 100644
--- a/jaxlib/rocm/BUILD
+++ b/jaxlib/rocm/BUILD
@@ -498,6 +498,7 @@ cc_library(
         "-fno-strict-aliasing",
     ],
     features = ["-use_header_modules"],
+    visibility = ["//visibility:public"],
     deps = [
         ":hip_vendor",
         "//jaxlib:ffi",
