diff --git a/jaxlib/rocm/BUILD b/jaxlib/rocm/BUILD
index 72f2ec6da..11f620658 100644
--- a/jaxlib/rocm/BUILD
+++ b/jaxlib/rocm/BUILD
@@ -477,6 +477,7 @@ nanobind_extension(
 
 py_library(
     name = "rocm_gpu_support",
+    visibility = ["//visibility:public"],
     deps = [
         ":_hybrid",
         ":_linalg",
@@ -528,6 +529,7 @@ cc_library(
 
 nanobind_extension(
     name = "rocm_plugin_extension",
+    visibility = ["//visibility:public"],
     srcs = ["rocm_plugin_extension.cc"],
     module_name = "rocm_plugin_extension",
     deps = [
