diff --git a/jaxlib/rocm/BUILD b/jaxlib/rocm/BUILD
index 0c2c2422a..4f0f323ec 100644
--- a/jaxlib/rocm/BUILD
+++ b/jaxlib/rocm/BUILD
@@ -537,6 +537,7 @@ nanobind_extension(
     srcs = ["rocm_plugin_extension.cc"],
     module_name = "rocm_plugin_extension",
     deps = [
+        ":hip_gpu_kernel_helpers",
         ":py_client_gpu",
         "//jaxlib:kernel_nanobind_helpers",
         "//jaxlib/gpu:gpu_plugin_extension",
diff --git a/jaxlib/rocm/rocm_plugin_extension.cc b/jaxlib/rocm/rocm_plugin_extension.cc
index 508aa0b51..ab3208756 100644
--- a/jaxlib/rocm/rocm_plugin_extension.cc
+++ b/jaxlib/rocm/rocm_plugin_extension.cc
@@ -23,6 +23,7 @@ limitations under the License.
 #include "jaxlib/gpu/gpu_plugin_extension.h"
 #include "jaxlib/gpu/py_client_gpu.h"
 #include "jaxlib/kernel_nanobind_helpers.h"
+#include "jaxlib/gpu/gpu_kernel_helpers.h"
 
 namespace nb = nanobind;
 
@@ -82,6 +83,13 @@ nb::dict FfiRegistrations() {
   return dict;
 }
 
+int ROCmDeviceCount(){
+  int device_count = -1;
+  JAX_THROW_IF_ERROR(JAX_AS_STATUS(hipInit(0)));
+  JAX_THROW_IF_ERROR(JAX_AS_STATUS(hipGetDeviceCount(&device_count)));
+  return device_count;
+}
+
 }  // namespace
 
 NB_MODULE(rocm_plugin_extension, m) {
@@ -107,5 +115,6 @@ NB_MODULE(rocm_plugin_extension, m) {
         return device_ordinal;
       },
       nb::arg("data_value"));
+  m.def("get_device_count", &ROCmDeviceCount);
 }
 }  // namespace jax
