diff --git a/third_party/py/python_init_rules.bzl b/third_party/py/python_init_rules.bzl
--- a/third_party/py/python_init_rules.bzl
+++ b/third_party/py/python_init_rules.bzl
@@ -38,12 +38,11 @@ def python_init_rules(extra_patches = []):
 
     tf_http_archive(
         name = "rules_python",
-        sha256 = "c85d5db38d3eac06167a13b10c9dba54b003a986cd4f1ebc00806b74e7c12f06",
-        strip_prefix = "rules_python-1.8.4",
-        urls = tf_mirror_urls("https://github.com/bazelbuild/rules_python/releases/download/1.8.4/rules_python-1.8.4.tar.gz"),
+        sha256 = "fa7dd2c6b7d63b3585028dd8a90a6cf9db83c33b250959c2ee7b583a6c130e12",
+        strip_prefix = "rules_python-1.6.0",
+        urls = tf_mirror_urls("https://github.com/bazelbuild/rules_python/releases/download/1.6.0/rules_python-1.6.0.tar.gz"),
         patch_file = [
-            "@xla//third_party/py:rules_python_scope.patch",
+            "@xla//third_party/py:rules_python_pip_version.patch",
             "@xla//third_party/py:rules_python_freethreaded.patch",
-            "@xla//third_party/py:rules_python_versions.patch",
         ] + extra_patches,
     )
