diff --git a/third_party/py/python_init_rules.bzl b/third_party/py/python_init_rules.bzl
index d1b268f..dbb4b4b 100644
--- a/third_party/py/python_init_rules.bzl
+++ b/third_party/py/python_init_rules.bzl
@@ -37,12 +37,10 @@ def python_init_rules(extra_patches = []):
 
     tf_http_archive(
         name = "rules_python",
-        sha256 = "c85d5db38d3eac06167a13b10c9dba54b003a986cd4f1ebc00806b74e7c12f06",
-        strip_prefix = "rules_python-1.8.4",
-        urls = tf_mirror_urls("https://github.com/bazelbuild/rules_python/releases/download/1.8.4/rules_python-1.8.4.tar.gz"),
+        sha256 = "8964aa1e7525fea5244ba737458694a057ada1be96a92998a41caa1983562d00",
+        strip_prefix = "rules_python-1.8.5",
+        urls = tf_mirror_urls("https://github.com/bazelbuild/rules_python/releases/download/1.8.5/rules_python-1.8.5.tar.gz"),
         patch_file = [
-            "@xla//third_party/py:rules_python_scope.patch",
-            "@xla//third_party/py:rules_python_freethreaded.patch",
-            "@xla//third_party/py:rules_python_versions.patch",
+            "@xla//third_party/py:rules_python_pip_version.patch",
         ] + extra_patches,
     )
